portal_objects/reference_genome.yaml

```diff
@@ -0,0 +1,12 @@
+################################################################
+# GRCh38 Genome Reference
+################################################################
+name: GRCh38
+version: GCA_000001405.15
```

Comment:
Don't currently have a version on the ReferenceGenome item, but can add it to an existing PR.